Task10_Auth/src/main/java/AuthSingleton.java

```diff
@@ -0,0 +1,73 @@
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+import reseponsesDTO.TokenDto;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.Properties;
+
+public class AuthSingleton {
+    private static AuthSingleton object;
+    private static String writeToken;
+    private static String readToken;
```

Comment:
Do not see reasons to make these 2 fields static

Reply:
fixed.